services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/cache/NodeTopicCache.java

```diff
@@ -0,0 +1,44 @@
+package com.futurewei.alcor.dataplane.cache;
```

Comment:
@VanderChen A quick comment: Pls add MIT license statement in every new file in this PR. You can check the existing file for reference. 

@yanmo96 Let us see if we could figure out automatic way of license statement insertion so that our dev doesn't need to worry about license statement.

Reply:
I didn't notice the licence, IntelliJ hides the license notice and the inports. I will watchout for this and change these files as necessary.

Reply:
> I didn't notice the licence, IntelliJ hides the license notice and the inports. I will watchout for this and change these files as necessary.

Thanks for watching this out.